### PR TITLE
doc: usb: Update documentation of CDC ACM regarding UART API

### PR DESCRIPTION
**Pull request by aelray**

### CODE REVIEW

**Review by carlescufi — DISMISSED** [review]

**Comment by carlescufi** [comment]
@aelray could you please fix the compliance checks?

File: doc/services/usb/uds_cdc_acm.rst [hunk 1]
```diff
@@ -14,6 +14,8 @@ But there are two important differences in behavior to a real UART controller:
   until then any data is discarded
 * If device is connected to the host, it still needs an application
   on the host side which requests the data
+* Asynchronous UART API is not compatible, only interrupt or polling UART API
```
Comment (jfischer-no) [inline doc/services/usb/uds_cdc_acm.rst:17]: Asynchronous UART API is not **supported**, ...

Comment (aelray) [reply doc/services/usb/uds_cdc_acm.rst]: fixed

**Comment by aelray** [comment]
> @aelray could you please fix the compliance checks?

done

**Review by jfischer-no — COMMENTED** [review]
Commit title and message should be like:
```
doc: usb: update documentation of CDC ACM regarding UART API

Add comment that asynchronous UART API is not supported.
```

**Comment by aelray** [comment]
> Commit title and message should be like:
> 
> ```
> doc: usb: update documentation of CDC ACM regarding UART API
> 
> Add comment that asynchronous UART API is not supported.
> ```

Done

**Review by jfischer-no — APPROVED** [review]

**Comment by aelray** [comment]
> @aelray could you please fix the compliance checks?

Hopefully done as well

**Pull request closed without merging**